cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -150,7 +162,13 @@ def index_item(item, skip_index=False):
                 skip_child_index = skip_index or \
                     (triggered_at is not None and (triggered_at - item.subtree_edited_on) > reindex_age)
                 for child_item in item.get_children():
-                    index_item(child_item, skip_index=skip_child_index)
+                    temp_content = getattr(item, 'content_groups', None)
+                    if temp_content and temp_content is not []:
```

Comment:
I think `temp_content is not []` should be replaced with `temp_content != []` if you wish to check that `temp_content` doesn't have an empty list for its value.